server/grpc/runner/runner.go

```diff
@@ -139,9 +143,15 @@ func (s *Server) Result(ctx context.Context, req *ResultRequest) (*ResultRespons
 	if err != nil {
 		return nil, err
 	}
+	start, ok := s.inProgress.Load(req.ExecutionHash.String())
+	if !ok {
+		panic("execution should be in memory")
+	}
 	msg := executionmodule.MsgUpdate{
 		Executor: acc.GetAddress(),
 		Hash:     req.ExecutionHash,
+		Start:    start.(uint64),
+		Stop:     uint64(time.Now().UnixNano()),
```

Comment:
why not just keeping the elapsed time?

Reply:
I don't really like to keep only the elapsed time because we lose some information. Having the duration is a thing but knowing when it started (or stopped) is good too if we want to debug what happened.
We could have the duration but in this case, we also need to have the `startTime` to make sure that we can calculate everything again.
In general, I prefer to have raw data here that can be used to calculate and not just have the calculation result that for some reasons may be wrong.

Reply:
As discussed, we agreed on keeping start and stop as we will always need 2 out of the 3 parameters (start, stop, duration) in order to calculate anything.